README.md

```diff
@@ -14,45 +14,42 @@ The toolkit library jar is hosted on https://esri.jfrog.io/artifactory/arcgis.
 
 To add the dependency to your project using Gradle:
 ```groovy
-apply plugin: 'application'
-
-// Runtime SDK dependency
-apply plugin: 'com.esri.arcgisruntime.java'
-buildscript {
-    repositories {
-        maven {
-            url 'https://esri.jfrog.io/artifactory/arcgis'
-        }
-    }
-    dependencies {
-        classpath 'com.esri.arcgisruntime:gradle-arcgis-java-plugin:1.0.0'
-    }
+plugins {
+    'application'
 }
-arcgis.version = '100.2.1'
 
-// Toolkit dependency
+// Replace with version number of ArcGIS SDK you are using in your app, such as:
+// arcgisVersion = '100.11.0'. See table below for SDK Versions that support the toolkit.
+ext {
+  arcgisVersion = '100.2.1'
+}
+
+compileJava.options.encoding = 'UTF-8'
+
+// Toolkit and Runtime SDK repository
 repositories {
   maven {
       url 'https://esri.jfrog.io/artifactory/arcgis'
   }
 }
 
 dependencies {
-  compile 'com.esri.arcgisruntime:arcgis-java-toolkit:100.2.1'
+  implementation "com.esri.arcgisruntime:arcgis-java:$arcgisVersion"
+  implementation "com.esri.arcgisruntime:arcgis-java-toolkit:100.2.1"
```

Comment:
There are missing parts from the dependencies.  jnilibs and resources are not there.  The gradle script needs to be based on something which we have seen works.  This samples is a good example of this:

https://github.com/Esri/arcgis-runtime-samples-java/blob/master/map_view/scale-bar/build.gradle

Reply:
Done